monero-outreach-docs/translations/zh-hans/swot/swot_zh-hans.md

```diff
@@ -0,0 +1 @@
私有财富。门罗币的隐私特性使其对高净值个人和机构具有吸引力，这些人希望参与新兴的加密货币经济，同时保持财务隐私。

New developers. Being the number three cryptocurrency in terms of developers, Monero has an opportunity to attract new talent and surpass the amount of developers as Bitcoin or Ethereum, the only two current leaders over Monero by number of developers.
新开发人员。作为开发者数量排名第三的加密货币，门罗币有机会吸引新的人才，并超过比特币(Bitcoin)或以太坊(Ethereum)这两种目前在开发者数量上领先于门罗币的货币。


Open Source Community. The free and open source ethos of Monero are very strong, which make it a candidate to be adopted by the ever expanding global open source community. Continued adoption can lead to an even stronger ecosystem and increased liquidity.
开源社区。门罗币的自由和开源精神非常强大，这使得它成为不断扩展的全球开源社区的备选项。继续采用门罗币可形成更强大的生态系统，并增加其流动性。

Artists. Artists of various genres have begun to integrate Monero payments, a trend that may continue and thus, support the use case of Monero.
艺术家。不同流派的艺术家已经开始整合门罗币支付，这一趋势可能会继续下去，从而支持门罗币的使用。

Threats 风险

Regulation. Monero faces a threat of being removed from exchanges in the future if regulation around exchanges continues. The privacy focus of Monero also makes it a potential target of KYC/AML regulation. Although the ability to share private view keys for auditing Monero will curb this threat substantially, it could create near-term challenges until regulators are educated.
监管。如果围绕交易所的监管继续下去，门罗币未来将面临被逐出交易所的风险。门罗币对隐私的关注也使其成为客户身份验证/反洗钱监管的潜在目标。尽管门罗币共享审计的查阅私钥可以在很大程度上遏制这一风险，但在监管机构弄明白以前，这可能会在短期内带来挑战。

Market share. Monero could lose market share to other privacy coins.
市场份额。门罗币的市场份额可能会被其他隐私币夺走。

Hidden inflation. While this risk has been asserted to be provably irrational many times by prominent and trusted community members and developers, it is still a theoretical risk.
隐性通货膨胀。虽然卓越的、值得信任的社区成员和开发人员已经多次断言这个风险是不合理的，但它仍然是一个理论上的风险。

Blockchain analysis. Various blockchain analysis risks exist, including timing attacks, poisoned outputs, clearnet usage (unobfuscated IP address), and honeypots. Fortunately the community of developers discuss these regularly and are actively working to address the risks. Monero’s workgroups also develop videos, articles, and best practice PSAs （PSA - Public Service Announcement）to help new users learn these threats and how to remain as safe as possible.  
区块链分析。存在各种各样的区块链分析风险。幸运的是，开发人员社区会定期讨论这些问题，并积极地处理这些风险。门罗币的工作组还开发视频、文章和最佳实践公益宣传，以帮助新用户了解这些风险以及如何尽可能保持安全。

Network Attacks. A variety of attacks on the Monero network are theoretically possible and have been explored in depth by the Breaking Monero series.
网络攻击。对门罗币网络的各种攻击在理论上是可能的，并已经在攻击门罗币系列被深入探讨过。

Misinformation and censorship. Coordinated misinformation campaigns could potentially lead to attempts to ban Monero in certain jurisdictions or decrease its popularity in general. While Monero cannot technically be censored, this theoretical threat could hurt its adoption and decrease liquidity in certain jurisdictions.

错误信息和审查。协调错误信息事件可能导致在某些司法管辖区禁止门罗币或降低其总体受欢迎程度。虽然门罗币在技术上不可能受到审查，但这种理论上的风险可能会影响其在某些司法管辖区的采用，并降低流动性。

```

Comment:
This file is also missing in the 'en' files, I will also add it today. The file also needs to be changed of folder:

- Move to folder _/zh-hans/storyboard_

**The files name is okay, I will add the 'en' version with the same name.**

Reply:
Done.